modules/trusted-firmware-m/Kconfig

```diff
@@ -141,6 +141,11 @@ config TFM_CRYPTO_PARTITION_STACK_SIZE
 	help
 	  The stack size of the Crypto Secure Partition
 
+config TFM_CRYPTO_NV_SEED
+	# Hidden option to set value based on TF-M minimal profile
```

Comment:
We have generally given this type of information in the help-text instead of completely hiding it with a comment

help
    Prompt-less configuration controlling NV Seed support.
    Not set when TFM_PROFILE_TYPE_MINIMAL is used

Reply:
Added help text